script.js

```diff
@@ -1,4 +1,112 @@
+var die1;
+var die2;
+var player1Score = [];
+var player1GlobalScore = 0;
+var player2Score = [];
+var player2GlobalScore = 0;
+var player = 1;
+var mode = "dice roll";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var outputMessage;
+  while (player == 1) {
+    if (mode == "dice roll") {
+      var diceResult = roll2Dice();
+      mode = "choose dice order";
+      outputMessage = `Welcome Player 1. <br>
+     ${diceResult} <br>Choose the order of the dice.`;
+    } else if (mode == "choose dice order") {
+      var player1Number = concatDiceResult(input);
+      player1Score.push("" + player1Number);
+
+      // sum of player 1's score thus far
+      var index = 0;
+      var player1CurrentScore = 0;
+      while (index < player1Score.length) {
+        var currentLoopScore = player1Score[index];
+        player1CurrentScore =
+          Number(player1CurrentScore) + Number(currentLoopScore);
+        index = index + 1;
+      }
+      player1GlobalScore = player1CurrentScore;
```

Comment:
This could be abstracted into a helper function as this code seems to be repeated below